doc/src/vpr/file_formats.rst

```diff
@@ -1046,3 +1046,158 @@ An example of what a generated routing resource graph file would look like is sh
         </rr_edges>
     </rr_graph>
 .. _end:
+
+Block types usage summary (.txt .xml or .json)
+-----------------------------------------
+
+Block types usage summary is a file written in human or machine readable format.
+It describes types and the amount of FPGA resources that are used by implemented design.
```

Comment:
Should say cluster-level FPGA resource that are used ...

Reply:
Fixed